src/LoginControl.test.js

```diff
@@ -42,7 +42,6 @@ it("renders LoginControl with LogoutButton component when isAuthenticationInfoVa
   const wrapper = shallow(
     <LoginControl isAuthenticationInfoValid={mockIsAuthenticationInfoValid} />
   );
-
```

Comment:
Losing empty lines. This makes it less readable. Revert this file.

Reply:
Reverted.